tests/tests.js

```diff
@@ -4,42 +4,145 @@ const employee = require('./employee')
 const products = require('./products')
 const pricing = require('../pricing')
 
-describe('calculateProductPrice', () => {
-  it('returns the price for a voluntary life product for a single employee', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee'],
-      coverageLevel: [{ role: 'ee', coverage: 125000 }],
-    }
-    const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
-
-    expect(price).to.equal(39.37)
+describe('Pricing', () => {
+
+  describe('calculateLTDPrice', () => {
+    it('returns price for LTD plan for a single employee', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+        coverageLevel: [
+          { role: 'ee', coverage: 125000 }
+        ],
+      }
+      const price = pricing.calculateLTDPrice(products.longTermDisability, employee, selectedOptions)
+
+      expect(price).to.equal(32.04)
+    })
+
+    it('returns the price of an LTD plan without employer contribution', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+      }
+      const price = pricing.calculateLTDPrice(products.longTermDisability, employee, selectedOptions)
+
+      expect(price).to.equal(32.04)
+    })
+
```

Comment:
These tests are testing the same thing. LTD doesn't use coverage levels so removing it in the second test doesn't change anything. Also both tests are for "without employer contribution" since the calculateLTDPrice code never looks at the employer contribution that is handled after by the calculateProductPrice function.